marklogic-unit-test-client/src/test/java/com/marklogic/test/unit/SampleParameterizedTest.java

```diff
@@ -34,12 +34,20 @@ public static void releaseDatabaseClient() {
     @ParameterizedTest
     @ArgumentsSource(SampleParameterizedTest.class)
     public void test(TestModule testModule) {
-        TestSuiteResult result = testManager.run(testModule);
-        for (TestResult testResult : result.getTestResults()) {
-            String failureXml = testResult.getFailureXml();
-            if (failureXml != null) {
-                fail(String.format("Test %s in suite %s failed, cause: %s", testResult.getName(), testModule.getSuite(), failureXml));
-            }
+    TestSuiteResult result = testManager.run(testModule);
```

Comment:
Also missing a brace in this class - I had to add one, wasn't compiling otherwise.